### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by slimbuck**
This PR:
- generates text versions of the colmap model so we can extract cameras
- removes 'magick' from the resize command because we don't use the magick build of imagemagic.

### CODE REVIEW

**Review by willeastcott — APPROVED** [review]

**Review by gdudovitch-sc — APPROVED** [review]
LGTM

File: convert.py [hunk 1]
```diff
@@ -175,21 +185,21 @@ def concat_alpha(seg_path):
 
         destination_file = os.path.join(args.source_path, "images_2", file)
         shutil.copy2(source_file, destination_file)
-        exit_code = os.system(magick_command + " mogrify -resize 50% " + destination_file)
```
Comment (isumygin-sc) [inline convert.py:178]: why explicit command instead of magick_command? I think original code handles it well

Comment (isumygin-sc) [reply convert.py:178]: after merging this https://github.sc-corp.net/Snapchat/playcanvas-monorepo/pull/2292 it should be fine

Comment (slimbuck) [reply convert.py:178]: The `magick mogrify` way of invoking imagemagick is when you're using the appImage-based single application install of imagemagick. We're using the packaged version, which requires invoking mogrify/convert etc directly. (see https://imagemagick.org/script/download.php)

**Review by isumygin-sc — APPROVED** [review]

**Pull request merged**